src/networking.c

```diff
@@ -109,6 +109,9 @@ static void clientSetDefaultAuth(client *c) {
     c->user = DefaultUser;
     c->authenticated = (c->user->flags & USER_FLAG_NOPASS) &&
                        !(c->user->flags & USER_FLAG_DISABLED);
+    if (c->authenticated) {
```

Comment:
not sure do we need check if the client is authenticated, since it already point to the `DefaultUser`.